internal/ct/handlers.go

```diff
@@ -319,6 +325,8 @@ func addChainInternal(ctx context.Context, opts *HandlerOptions, log *log, w htt
 		return http.StatusBadRequest, nil, fmt.Errorf("failed to parse add-chain contents: %s", err)
 	}
 
+	notBeforeAge.Record(ctx, time.Since(chain[0].NotBefore).Seconds())
```

Comment:
Given that this happens before we verify the chain - on purpose since this information will be useful to have insights into rate limiting, this won't provide information about what actually gets into the log, which could be misleading. How about we either:
 1. add an "accepted / rate_limited" field
 2. tag this metric with something like `tesseract.notbefore.age.unverified`

I think I have a slight preference for 1, but I'm keen to hear what you think!

Reply:
(2) LGTM!